src/main/groovy/com/palantir/gradle/javadist/DistributionExtension.groovy

```diff
@@ -62,6 +65,16 @@ class DistributionExtension {
         this.checkArgs = checkArgs.toList()
     }
 
+    public void customClasspath(String... customClasspath) {
+        assert customClasspath.every { isAllowedClasspathChar(it) }
```

Comment:
If I use the custom classpath to filter against the list of jars originally used in the LaunchConfigTask, do I still need to check for special characters?